swipe/src/main/kotlin/me/saket/swipe/SwipeableActionsBox.kt

```diff
@@ -51,7 +51,7 @@ fun SwipeableActionsBox(
   startActions: List<SwipeAction> = emptyList(),
   endActions: List<SwipeAction> = emptyList(),
   swipeThreshold: Dp = 40.dp,
-  backgroundUntilSwipeThreshold: Color = Color.DarkGray,
+  backgroundUntilSwipeThreshold: Color = Color.Transparent,
```

Comment:
I'd leave this as-is, mostly because this would be a breaking change for existing usages.